cli/cmd/install-cni-plugin.go

```diff
@@ -127,7 +127,7 @@ assumes that the 'linkerd install' command will be executed with the
 		&options.useWaitFlag,
 		"use-wait-flag",
 		options.useWaitFlag,
-		"Configures the CNI plugin to use the \"-w\" flag for the iptables command.")
+		"Configures the CNI plugin to use the \"-w\" flag for the iptables command. Default value is false")
```

Comment:
Nit: the standard we follow in other commands is to have the default in parenthesis
`Configures the CNI plugin to use the \"-w\" flag for the iptables command (default false)`

Reply:
Thanks for letting me know about the standard. I've updated the description